README.md

```diff
@@ -16,66 +16,30 @@ This repo is versioned with [Lerna](https://github.com/lerna/lerna).
-
-### Importing Styles
-
-The design system does not compile styles and templates. It is up to the product to do this. This generally means updating your webpack (grunt, gulp, etc) config. You can use the [storybook config](https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js) as an example, and further documentation in the [styles readme](https://github.com/NYPL/nypl-design-system/blob/development/styles/README.md).
+To use the Design System in your project:
 
-Make sure to include `glob-importer` in your (sass-loader)[https://github.com/NYPL/nypl-design-system/blob/development/storybook/.storybook-react/webpack.config.js#L2].
+1. Install the design system `npm install @nypl/design-system-react-components`
+2. Import the styles in your main css file `@import '~@nypl/design-system-react-components/dist/app.css';`
+3. Consult Storybook for the list of available components and props that they require.
+4. Usage example
 
-### Importing React Components
+```
+import * as DS from '@nypl/design-system-react-components';
 
-Once the Design System Styles package is imported, follow the steps in the React Components [README](https://github.com/NYPL/nypl-design-system/blob/development/src/react-components/README.md) to import and use the React Components.
+function NewComponent(props) {
+  return <DS.Link href="https://www.hathitrust.org/">HathiTrust</DS.Link>;
+}
+```
 
 ### Storybook
 
-There should be no need to run the static Storybook instances. Following the instructions above, the two development Storybook instances can run on a server that tracks changes. If the static Storybook npm script is run, make sure to not commit the `storybook/storybook-static` directory.
+There should be no need to run the static Storybook instances. If the static Storybook npm script is run, make sure to not commit the `storybook/storybook-static` directory.
 
-When change are merged into `master` (currently using `development`), Travis CI will automatically build the two Twig and React Storybook instances as static sites and deploy them to the `gh-pages` branch. If this needs to be tested locally, run `npm run storybook:static` and then view the `storybook/index.html` file in a browser. No server needs to be running since this is a static web site.
+When change are merged into `master` (currently using `development`), Travis CI will automatically build the Storybook instance as a static site and deploy to the `gh-pages` branch. If this needs to be tested locally, run `npm run storybook:static` and then view the `storybook/index.html` file in a browser. No server needs to be running since this is a static web site.
```

Comment:
Nit: changes

Re: branches, are we switching up our branching strategy? I'm fine with it but maybe we should describe current state and leave out future state until we're ready to make the change.